src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,405 @@
+            sub-001_task-rest_run-1_space-fsLR_res-2_den-10k_bold.dtseries.nii
+            sub-001_task-rest_run-1_space-fsLR_res-2_den-41k_bold.dtseries.nii
+            sub-001_task-rest_run-1_space-fsLR_bold.json
+```
+
+And the corresponding `sub-001_task-rest_run-1_space-fsLR_bold.json` file:
+
+```JSON
+{
+    "SkullStripped": true,
+    "Resolution": {
+        "1": "Matched with MNI152NLin6Asym 1.6mm isotropic",
+        "2": "Matched with MNI152NLin6Asym 2.0mm isotropic"
+    },
+    "Density": {
+        "10k": "10242 vertices per hemisphere (5th order icosahedron)",
+        "41k": "40962 vertices per hemisphere (6th order icosahedron)"
+    }
+}
+```
+
+## Masks
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat|func|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>][_desc-<label>]_mask.nii.gz
+```
+
+A binary (1 - inside, 0 - outside) mask in the space defined by `<space>`.
+If no transformation has taken place, the value of `space` SHOULD be set to `orig`.
+If the mask is an ROI mask derived from an atlas, then the `label` entity SHOULD
+be used to specify the masked structure
+(see [Common image-derived labels](#common-image-derived-labels)),
+and the `Atlas` metadata SHOULD be defined.
+
+JSON metadata fields:
+
+| **Key name** | **Description**                                                                                                                                |
+| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| RawSources   | Same as defined in [Introduction][intro], but elevated from OPTIONAL to REQUIRED                                                               |
+| Type         | RECOMMENDED. Short identifier of the mask. Reserved values: `Brain` - brain mask, `Lesion` - lesion mask, `Face` - face mask, `ROI` - ROI mask |
+| Atlas        | OPTIONAL. Which atlas (if any) was used to generate the mask. RECOMMENDED if `label` entity is defined.                                        |
+| Resolution   | REQUIRED if `res` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the resolution keyword.         |
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.            |
+
+Examples:
+
+```Text
+func_loc/
+    sub-001/
+        func/
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.nii.gz
+           sub-001_task-rest_run-1_space-MNI305_desc-PFC_mask.json
+```
+
+```Text
+manual_masks/
+    sub-001/
+        anat/
+            sub-001_desc-tumor_mask.nii.gz
+            sub-001_desc-tumor_mask.json
+```
+
+## Segmentations
+
+A *segmentation* is a labeling of a spatial image with anatomical structures
```

Comment:
Perhaps the "anatomical" qualifier could be removed to allow for a more generic definition of segmentation?

Reply:
I'm open to this, but I'm not sure "structures" evokes the same sense as "anatomical structures". What about something like:

```suggestion
A *segmentation* is a labeling of regions of an image
```

We could add an additional sentence to make clear that regions do not have to be defined anatomically:

```Markdown
Labeled regions may include anatomical structures (such as tissue class,
Brodmann area or white matter tract), discontiguous, functionally-defined
networks, or <maybe a third example?>.
```

What do you think?

Reply:
adding the sentence i think is good. you could add tumors and lesions perhaps.

Reply:
Yes, looks good to me, thanks.